.gitignore

```diff
@@ -24,4 +24,6 @@
 hack/tools/bin
 
 junit-report.xml
-/artifacts
\ No newline at end of file
+/artifacts
+
+examples/kcp/bin/*
```

Comment:
nit: technically this could be in `examples/kcp/.gitignore` avoiding rebase conflict.